test/unit/commands/project.test.js

```diff
@@ -10,7 +10,7 @@ const delStub = sinon.stub();
 const printSpy = sinon.spy();
 let callback;
 
-const program = proxyquire('../../../lib/commands/dataset', {
```

Comment:
This test was not passing for me locally. I'd like to suggest setting up protected branches to avoid repeating this mistake in the future.